services/ui-src/src/components/export/ExportedOverlayModalReportSection.test.tsx

```diff
@@ -0,0 +1,83 @@
+import { render, screen } from "@testing-library/react";
+import { axe } from "jest-axe";
+import { RouterWrappedComponent } from "utils/testing/setupJest";
+
+import { ExportedOverlayModalReportSection } from "./ExportedOverlayModalReportSection";
+
+import { EntityShape, FormField, OverlayModalPageShape } from "types";
+
+const section: OverlayModalPageShape = {
+  entityType: "",
+  stepType: "",
+  stepName: "",
+  stepInfo: [],
+  hint: "",
+  verbiage: {
+    addEntityButtonText: "",
+    dashboardTitle: "",
+    countEntitiesInTitle: false,
+    tableHeader: "",
+    addEditModalHint: "",
+    intro: {
+      section: "",
+      subsection: undefined,
+      hint: undefined,
+      info: undefined,
+      title: undefined,
+      subtitle: undefined,
+      spreadsheet: undefined,
+      exportSectionHeader: undefined,
+    },
+  },
+  modalForm: {
+    id: "",
+    fields: [],
+  },
+  name: "",
+  path: "",
+};
+
+const entityStep: (string | FormField)[] = [
+  "mockType",
+  "mock plan name",
+  "mock hint",
+  { id: "mock-id", props: { label: "mock field 1" }, type: "", validation: "" },
+];
+
+const entity: EntityShape = {
+  id: "entity-id",
+  type: "initiative",
+  "mock-id": "mock value",
+  mockType: [
+    {
+      id: "step-entity-id",
+    },
+  ],
+};
+
+const testComponent = (
+  <RouterWrappedComponent>
+    <ExportedOverlayModalReportSection
+      section={section}
+      entityStep={entityStep}
+      entity={entity}
+    />
+  </RouterWrappedComponent>
+);
+
+describe("Test ExportedOverlayModalReportSection Component", () => {
+  beforeEach(() => {
+    render(testComponent);
+  });
+  test("Test ExportRETTable render", () => {
+    screen.debug();
+  });
```

Comment:
same as above: doesn't really prove it renders, does it? meaning, the test would pass the same if this was just a console.log(), right?

Reply:
Yea this wasn't supposed to be there lol